symfony/routing/5.1/config/packages/routing.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+    router:
+        utf8: true
+
+        # Replace these nodes to configure the "request context" to use to generate URLs
+        # in non-HTTP contexts, such as CLI commands.
+        # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
+        #context:
+        #    host: localhost
+        #    scheme: http
+        #    base_url: null
```

Comment:
shouldn't this be a single setting: `http://lcalhost/path/`?

Reply:
That's not how the router context works right now. Do you thinks it's worth adding such support?

Reply:
I think yes, that's how it should work to me.

Reply:
@benji07: interested?

Reply:
Tracked in https://github.com/symfony/symfony/issues/35121, implementation proposed in https://github.com/symfony/symfony/pull/35580